tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -0,0 +1,412 @@
+{
```

Comment:
<p>I think this might be overkill here. I think it would be better to just sample the final state and verify that it is what you expect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>